flexget/plugins/input/anidb_list.py

```diff
@@ -26,69 +28,100 @@ class AnidbList(object):
                 'error_pattern': 'user_id must be in the form XXXXXXX'},
             'type': {
                 'type': 'string',
-                'enum': ['shows', 'movies'],
+                'enum': ['shows', 'movies', 'ovas'],
                 'default': 'movies'},
+            'mode': {
+                'type': 'string',
+                'enum': ['all', 'undefined', 'watch', 'get', 'blacklist', 'buddy'],
+                'default': 'all'},
+            'pass': {
+                'type': 'string'},
             'strip_dates': {
                 'type': 'boolean',
-                'default': False}
+                'default': False},
+            'user_agent': {
+                'type': 'string',
+                'default': ('Mozilla/5.0 (Windows NT 10.0; Win64; x64) '
+                            'AppleWebkit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36')}
         },
         'additionalProperties': False,
         'required': ['user_id'],
         'error_required': 'user_id is required'
     }
 
+    def __mode_string_to_int(self, mode_string):
+        if mode_string == 'buddy':
+            return 11
+        return self.schema['properties']['mode']['enum'].index(mode_string)
```

Comment:
It's much cleaner to have it as a dict instead eg.

```
MODE_MAP = {
  'all': 0,
  'undefined': 1,
  'watch': 2,
  'get': 3, 
  'blacklist': 4, 
  'buddy': 11
}
```

Then the config schema simply becomes

```
            'mode': {
                'type': 'string',
                'enum': list(MODE_MAP.keys()),
                'default': 'all'
            },
```